makefile

```diff
@@ -382,7 +382,7 @@ rebuild-executor: build-docker-executor
 
 .ONESHELL:
 tests-e2e-teardown:
-	docker rm -f nats redis pulsar server executor postgres || true
+	docker rm -f nats redis pulsar server executor postgres lookout-ingester-migrate lookout-ingester jobservice || true
```

Comment:
this is tearing down services that are not started by the "-setup" target

Reply:
Good call.

Reply:
This is actually a bug fix that I notice tests-e2e-setup creates but does not teardown.  

But for this work, I should actually call teardown-localdev.